tools/ci_build/build.py

```diff
@@ -691,6 +693,10 @@ def run_subprocess(args, cwd=None, capture_stdout=False, dll_path=None, shell=Fa
                 my_env["LD_LIBRARY_PATH"] += os.pathsep + dll_path
             else:
                 my_env["LD_LIBRARY_PATH"] = dll_path
+    # Add nvcc's folder to PATH env so that our cmake file can find nvcc
+    if cuda_home:
+        my_env["PATH"] = os.path.join(cuda_home, "bin") + os.pathsep + my_env["PATH"]
```

Comment:
should we also check if `"PATH" in my_env` as is done for dll_path?

Reply:
PATH is always in env vars.